### PR TITLE
Fix linting issues raised by flake8

### PR DESCRIPTION
**Pull request by peymanslh**
I ran flake8 and fixed all issues raised by it.
In my next PR, I will add flake8 to pre-commit (I can't send that PR before this because pre-commit runs by tox and breaks CI)

The change in `isort` version is because of a bug that fixed in the latest release. [link](https://github.com/PyCQA/isort/releases/tag/5.12.0) - [issue link](https://github.com/PyCQA/isort/pull/2078)

### CODE REVIEW

File: tests/test_tablib_dbfpy_packages_utils.py [hunk 1]
```diff
@@ -151,7 +151,7 @@ def test_getDateTime_datetime_string(self):
 
         # Act / Assert
         with self.assertRaises(NotImplementedError):
-            output = utils.getDateTime(value)
+            _ = utils.getDateTime(value)
```
Comment (claudep) [inline tests/test_tablib_dbfpy_packages_utils.py:154]: Do we even need the attribution here?

**Review by claudep — CHANGES_REQUESTED** [review]
Minor things, but looks generally good, thanks!

File: tests/test_tablib_dbfpy_packages_utils.py (hunk 1 above)
Comment (hugovk) [inline tests/test_tablib_dbfpy_packages_utils.py:154]: Nope, fine like this:
```suggestion
            utils.getDateTime(value)
```

File: src/tablib/packages/dbfpy/fields.py [hunk 2]
```diff
@@ -256,8 +256,8 @@ def encodeValue(self, value):
             if 0 <= _ppos <= self.length:
                 _rv = _rv[:self.length]
             else:
-                raise ValueError("[%s] Numeric overflow: %s (field width: %i)"
-                    % (self.name, _rv, self.length))
+                raise ValueError("[{}] Numeric overflow: {} (field width: {})".format(
```
Comment (hugovk) [inline src/tablib/packages/dbfpy/fields.py:259]: Or f-string?

File: src/tablib/formats/_ods.py [hunk 3]
```diff
@@ -7,7 +7,11 @@
 from odf import opendocument, style, table, text
 
 bold = style.Style(name="bold", family="paragraph")
-bold.addElement(style.TextProperties(fontweight="bold", fontweightasian="bold", fontweightcomplex="bold"))
+bold.addElement(style.TextProperties(
+    fontweight="bold",
+    fontweightasian="bold",
+    fontweightcomplex="bold"
```
Comment (hugovk) [inline src/tablib/formats/_ods.py:13]: Trailing comma, so if we add another, only one line is in the diff:
```suggestion
    fontweightcomplex="bold",
```

File: src/tablib/packages/dbfpy/record.py [hunk 4]
```diff
@@ -220,12 +219,9 @@ def delete(self):
 
     def toString(self):
         """Return string packed record values."""
-#        for (_def, _dat) in zip(self.dbf.header.fields, self.fieldData):
-#
-
+        #  for (_def, _dat) in zip(self.dbf.header.fields, self.fieldData):
```
[the remark this answers is not in the document]
Comment (claudep) [reply src/tablib/packages/dbfpy/record.py:222]: Note that dbfpy is vendored code, we should make only minimal changes.

[the remark this answers is not in the document]
Comment (peymanslh) [reply src/tablib/packages/dbfpy/record.py:222]: There are some one-line commented codes like this:

https://github.com/jazzband/tablib/blob/b2be12e5b647111d8e0f24c8bab646a97a3d68f2/src/tablib/packages/dbfpy/fields.py#L357

What should I do about them?

File: tests/test_tablib_dbfpy_packages_utils.py (hunk 1 above)
Comment (peymanslh) [reply tests/test_tablib_dbfpy_packages_utils.py]: Thanks, fixed it.

Comment (peymanslh) [reply tests/test_tablib_dbfpy_packages_utils.py]: Thanks.

File: src/tablib/packages/dbfpy/fields.py (hunk 2 above)
Comment (peymanslh) [reply src/tablib/packages/dbfpy/fields.py]: Fixed!

File: src/tablib/formats/_ods.py (hunk 3 above)
Comment (peymanslh) [reply src/tablib/formats/_ods.py]: Fixed

**Comment by peymanslh** [comment]
@claudep @hugovk 
Thanks for your review, I applied your suggestion and squashed my commits.
Is there anything I need to change?

**Review by claudep — APPROVED** [review]
Thanks for your work!

**Pull request merged**